bitnami/spark/templates/statefulset-master.yaml

```
@@ -145,6 +145,10 @@ spec:
             - name: http
               containerPort: {{ .Values.master.webPort }}
               protocol: TCP
+            {{- if .Values.security.ssl.enabled }}
+            - name: https
+              containerPort: {{ .Values.master.webPortHttps }}
+            {{- end }}
```

For consistency, please add the `protocol` field as you did in the worker statefulset and as in the field above.
```suggestion
            {{- if .Values.security.ssl.enabled }}
            - name: https
              containerPort: {{ .Values.master.webPortHttps }}
              protocol: TCP
            {{- end }}
```